python/libsonata/__init__.py.in

```diff
@@ -1,3 +0,0 @@
-from ._libsonata_python import *
-
-__version__ = "${SONATA_VERSION}"
```

Comment:
where does the version string come from instead?

Reply:
Ah yes, I need to keep that.